app/Resources/translations/default/messages.xlf

```diff
@@ -2960,4 +2960,18 @@ Comment: $this->list_modules_categories['search_filter']['name'] = $this->l('Sea
       </trans-unit>
     </body>
   </file>
+  <file original="modules/ps_emailsubscription/ps_emailsubscription.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="2df860f82be52de57fc5e3f7ecac9431">
+        <source>Newsletter subscription: Unable to delete customer using email.</source>
+        <target>Newsletter subscription: Unable to delete customer using email.</target>
```

Comment:
This wording is nowhere to be found here: https://github.com/PrestaShop/ps_emailsubscription/blob/master/ps_emailsubscription.php... that's weird?

Reply:
In the project I used to manage the export of translations, I found it in the module in the `hookActionDeleteGDPRCustomer` hook

Reply:
FYI, both wordings can be found in the 1.7.4 catalog on Crowdin. But not in the following catalogs.